src/server_manager/install_scripts/install_server.sh

```diff
@@ -81,18 +94,59 @@ function log_for_sentry() {
 # Check to see if docker is installed.
 function verify_docker_installed() {
   if ! command_exists docker; then
-    log_error "Docker CE must be installed, please run \"curl -sS https://get.docker.com/ | sh\" or visit https://docs.docker.com/install/"
-    exit 1
+    log_error "FAILED"
+    echo -n "> Would you like to install Docker? [Y/n] "
```

Comment:
Maybe say: `"> Would you like to install Docker? This will run 'curl -sS https://get.docker.com/ | sh'. [Y/n]: "`

Reply:
Done in d281454.